src/org/zaproxy/zap/extension/pscanrulesBeta/resources/help/contents/pscanbeta.html

```diff
@@ -36,7 +36,12 @@
 
 <H2>CSRF Countermeasures</H2>
 The CSRFCountermeasures plugin identifies "potential" vulnerabilities with the lack of known CSRF 
-countermeasures in pages with forms.
+countermeasures in pages with forms.<br>
+Post 2.5.0 you can specify a comma separated list of identifiers in the 
+<code>rules.csrf.ignorelist</code> parameter via the Options 'Rule configuration' panel.
+Any FORMs with a name or ID that matches one of these identifiers will be ignored when scanning for missing anti CSRF tokens.
```

Comment:
`anti CSRF ` should be made consistent with whatever you settle on for the ascan help and messages entries.